docker-compose.connect-stack.yml

```diff
@@ -0,0 +1,34 @@
+version: '2.3'
+
+services:
+  openvpn-client:
+    environment:
+      - DESECSTACK_VPN_SERVER=openvpn-server
+    networks:
+      vpnconnect:
+  openvpn-server:
+    networks:
+      vpnconnect:
+  replicator:
+    environment:
+      - DESECSTACK_VPN_SERVER=desec.$DESECSTACK_DOMAIN
+      - DESEC_NS_E2E2=test mode
+  test-e2e2:
+    environment:
+      - DESECSTACK_E2E2_SECONDARY_NS=10.16.0.2
```

Comment:
What does this variable do?

Reply:
Defines the target host for e2e2 tests. In desec-stack this is set to nsmaster, here it points to dnsdist of desec-ns.